pyarbtools/instruments.py

```diff
@@ -1401,36 +1541,46 @@ def bin_pdw_file_builder(self, pdwList):
             (bytes): Binary data that contains a full PDW file that can
                 be downloaded to and played out of the UXG.
         """
+        #Include frequency phase coding block flag: 1 = yes, 0 = no
+        includeFpcBlock = 1
```

Comment:
Possibly.  The fpcblock flag could be removed entirely.  I included it to emulate the older behavior if needed.  Options would be to remove the flag check and always include fpc block or add an argument to pass the flag.  Do you have a preference?

Reply:
I think this might go deeper than just handling the flag. I now see that setting the flag results in calling the function to build the FPC block. The modular nature of Pyarbtools suggests that the FPC building function should be _external_ to the PDW building function so the two can be defined independently.
For now, let's leave it the way it is since it doesn't break anything and doesn't expose a new function for documentation. We can revisit the structure after the next update.